README.md

```diff
@@ -38,6 +38,101 @@ use them. Right now there are solutions for the following use cases:
 - handle offline situations and caching when using OkHttp (`cache` package)
 - generic token based authentication handling with OkHttp (`auth` package)
 
+## Quick start for simple networking in stanwood apps
+
+You can find an example [over here](https://github.com/stanwood/architecture_sample_github_android) (private for now).
+
+Add the following dependencies (replace versions here with current versions):
+
+```groovy
+def retrofit_version = '2.4.0'
+implementation "com.squareup.retrofit2:retrofit:$retrofit_version"
+implementation "com.squareup.retrofit2:converter-gson:$retrofit_version"
+implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofit_version"
+
+implementation "com.google.code.gson:gson:2.8.2"
+
+implementation 'com.nytimes.android:store3:3.1.0'
```

Comment:
are we going for this ?

Reply:
Whoops, added that by mistake, will remove.